javascript/src/main/components/Footer/AppFooter.js

```diff
@@ -3,10 +3,17 @@ import React from "react";
 const AppFooter = () => {
   return (
     <footer className="bg-light p-3 text-center">
-      This is a demo of a minimal React frontend + Spring Boot backend app.
-      Check out the source code on{" "}
-      <a href="https://github.com/ucsb-cs156-f20/demo-spring-nextjs-app">
-        GitHub
+      This app is a class project of {" "}
+      <a href="https://ucsb-cs156.github.io">
+        CMPSC 156
+      </a>
+       at {" "}
+      <a href="https://ucsb.edu">
```

Comment:
Here is a comment about this line of code.

I wonder whether it would be better, if this line of code were written like this...